packages/esm-patient-medications-app/src/order-basket/order-basket-search.scss

```diff
@@ -15,3 +15,7 @@
   box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.3);
   z-index: 1;
 }
+
+.searchPopupContainer :global(.bx--search-input:focus) {
+  outline: 2px solid #ff832b;
```

Comment:
What's that color?

Reply:
It's a flavour of orange. It's not in `esm-core` yet, I'd planned to add it in a follow up PR and update this.

<img width="556" alt="Screenshot 2021-09-12 at 13 11 54" src="https://user-images.githubusercontent.com/8509731/132983674-4a12eeda-30b6-4ef0-a877-f706f81a1b36.png">

Reply:
It should be added. In general, all used colors should be properly defined by the designer. We should not need to add new colors - the palette should be settled by now.